### ISSUE TITLE
[Feature] Validation_pipeline is the copy of training_pipeline?

### ISSUE THREAD

**Issue by EricLina** [hao-ai-lab/FastVideo#805]
### Motivation

Hello, I found that in `fastvideo/training/wan_training_pipeline.py`, you copy validation_pipeline directly, that means you need two `WanPipeline` while training.
Could you add a option like `self.training` for the evaluation mode?
